src/components/tasks/card/index.tsx

```diff
@@ -29,42 +24,56 @@ const Card: FC<Props> = ({ content }) => {
 
   const localStartedOn = new Date(parseInt(startedOn, 10) * 1000);
   const fromNowStartedOn = moment(localStartedOn).fromNow();
-  const readableStartedOn = `${localStartedOn.toLocaleDateString()}, ${localStartedOn.toLocaleTimeString()}`;
 
   const localEndsOn = new Date(parseInt(endsOn, 10) * 1000);
   const fromNowEndsOn = moment(localEndsOn).fromNow();
-  const readableEndsOn = `${localEndsOn.toLocaleDateString()}, ${localEndsOn.toLocaleTimeString()}`;
+
+  const statusFontColor = status === 'active' || 'assigned' || 'completed' || 'pending' ? '#00a337' : '#f83535';
```

Comment:
Can we use constants here please?

Reply:
done 
![image](https://user-images.githubusercontent.com/45538392/116029165-4d68d600-a676-11eb-83a8-69c8e498b04d.png)

Reply:
its better if we name the constant all uppercase (good practise)
```javascript
const ACTIVE = 'active'
```

Reply:
Also can we please move the constants in a separate file?

Reply:
That will done. Thank you for letting me know the best practices 😃